docs/source/install-django-project.rst

```diff
@@ -22,161 +22,120 @@ downloading data as a researcher.
 
-These instructions will have you create and start a RabbitMQ image in Docker.
-
-Install and start rabbitmq via docker:
+   .. code-block:: shell
 
-.. code-block:: shell
+      make serve
 
-   docker run -d --name lookit-rabbit --env-file .env -p 5672:5672 rabbitmq:3.8.16-management-alpine
-   docker cp ./rabbitmq.sh lookit-rabbit:/rabbitmq.sh
-   docker exec -it lookit-rabbit /bin/sh -c "sh /rabbitmq.sh"
+   Once the services are up and worker has exited due to a permission constraint, we'll set the permissions in the container and restart the worker:
 
-Postgres
-~~~~~~~~
+   .. code-block:: shell
 
-This is covered above, but as sometimes databases can be ephemeral during development I felt that it deserved its own section.
+      make broker-perms
 
-Create a Postgres database using the following command:
-   
-.. code-block:: shell
+   From time to time, the container will need to be recreated,  when this happens you may need to run "broker-perms" again. 
 
-   poetry run create-db
+At this point, the services should all be up and running.  Direct your browser to https://localhost:8000 to see the local environment. In the future, to start the services you will only need to run "make serve"
 
-To reset the database:
 
-.. code-block:: shell
+Django
+~~~~~~
 
-   poetry run reset-db
+Here are few Django related tasks that might come up every now and then. 
 
-To reset the database and load an existing sql data file:
+To migrate the existing database:
 
 .. code-block:: shell
 
-   poetry run reset-db -s /location/of/sql/file
-
+   make migrate
 
-Create Superuser
-~~~~~~~~~~~~~~~~
-
-You can create a user through UI or if you'd rather have Superuser access you can create a user with the manage script.
-  
-Create a superuser by running:
+If you need to create an entry in the Site table:
 
 .. code-block:: shell
 
-   poetry run ./manage.py createsuperuser
-      
-Now you should be ready for anything. Going forward, you can run the server using the 
-directions below.
-
-Running the server
-~~~~~~~~~~~~~~~~~~~
-
-Again, this is covered above, but there is a case you'd need to run the development server with SSL.  This section will cover both variants. 
+   make site
 
-To run the Lookit server locally, run:
+To create a superuser:
 
 .. code-block:: shell
 
-   poetry run ./manage.py runserver
+   make superuser
 
-Or to use the invoke script:
+To run tests:
 
 .. code-block:: shell
 
-   poetry run invoke server
+   make tests
 
-Now you can go to http://localhost:8000 to see your local Lookit server! You should be able to log in using 
-the superuser credentials you created during setup.
 
-To view the HTTPS version of the local development add the ``https`` argument to the above command:
+Rabbitmq
+~~~~~~~~
+
+The broker should come up with the rest of the Docker services.  If you get a Celery worker error due to permissions, you can run the following command to resolve the issue and restart the worker service:
```

Comment:
Might be useful to say what the Celery worker error looks like, e.g. with a screenshot or text.